internal/perms/grants.go

```diff
@@ -235,6 +235,9 @@ func (g *Grant) unmarshalJSON(ctx context.Context, data []byte) error {
 			if !ok {
 				return errors.New(ctx, errors.InvalidParameter, op, fmt.Sprintf("unable to interpret %q element %q as string", "ids", id))
 			}
+			if idStr == "" {
+				return errors.New(ctx, errors.InvalidParameter, op, `segment "ids=" not formatted correctly, missing value`)
```

Comment:
Can this be more specific, as in the error on line 236? "ids" isn't missing a value, some id within it is...